gpkit/solution_array.py

```diff
@@ -80,17 +80,7 @@ def sens(self, nomial):
 
```

Comment:
should the docstring be updated? What are the valid input types for `nomial`? Variable? Monomial? Signomial?